flow/metrics.yaml

```diff
@@ -30,6 +35,13 @@ components:
             Latency metrics.
           $ref: '#/components/schemas/Flow.Latency.Metrics'
           x-field-uid: 4
+        rx_per_port:
```

Comment:
should we also have `rx_per_lag` and `rx_per_vxlan`? Or maybe if we want to support those in future, shall we do `rx_per_connection`?

Reply:
Currently we do not have support for drill down stats for lag or vxlan in TE. So we will support ports level disaggregation for now. But the model is very extensible and when we have the underlying support for those, we can extend the model without any breaking change.

Reply:
could you give an example of how the model would be extended? it would be good to see how that would be aligned with `rx_per_port` parameter. i'm thinking maybe we should use `choice` here and set it as `port` first and then extend to `lag` and `vxlan` later?

Reply:
I have restructured this part, now all additional tracking will come under "predefined_metric_tags" and we can add new properties under this. Current property "rx_name" will apply for both raw port and lag.